docs/Tutorials/Quorum-Dev-Quickstart/Getting-Started.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Quorum Dev Quickstart network setup quickstart command line tool
+---
+
+# Quorum Dev Quickstart
+
+[Quorum Dev Quickstart](https://github.com/ConsenSys/quorum-dev-quickstart) is a command line tool that allows
+users to set up a development GoQuorum network on their local machine in less than 2 minutes.
+
+![Quorum Dev Quickstart terminal demo](../../images/quickstart/quorum-dev-quickstart.gif)
+
+## Using Quorum Dev Quickstart
+
+GoQuorum Wizard is written in Javascript and designed to be run as a global NPM module from the command line.
+Make sure you have [Node.js/NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) installed
+(version 14 or higher).
+
+Using npx to run the quickstart without the need to install:
+
+```sh
+npx quorum-dev-quickstart
+```
+
+You can also install the wizard globally with npm:
+
+```Bash
+npm install -g quorum-dev-quickstart
+
+# Once the global module is installed, run:
+quorum-dev-quickstart
+```
+
+Note: Many installations of npm don't have permission to install global modules and will throw an EACCES error.
+[Here is the recommended solution from NPM](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally)
+
+## Dependencies
+
+Here are the dependencies (in addition to Nodejs 14+) that are required depending on the mode that you run the wizard in:
+* [Docker and Docker-compose](https://docs.docker.com/compose/install/)
+* [Truffle](https://www.trufflesuite.com/truffle) development framework
+* [curl command line](https://curl.haxx.se/download.html)
+* [MetaMask](https://metamask.io/)
+
+
+## Options
```

Comment:
Question: Is it preferable to document quickstart details in the official documentation or on the quickstart repository?

I am asking because keeping the official doc aligned with the quickstart is more cumbersome than just updating the repo doc when changing some bits of the quickstart.

Reply:
Also could avoid some duplication between Besu and GoQuorum docs.

Reply:
Yes, this is a good idea. Maybe we can put this in here in github pages or something?

Reply:
I think our best option is to keep the quickstart docs in one of the two places it currently exists and link to it from the other place. We have quickstart docs on:

- The doc site, which you added in this PR: <https://pegasys-docgoquorum--113.com.readthedocs.build/en/113/Tutorials/Quorum-Dev-Quickstart/Getting-Started/>
- The quickstart repo readme: <https://github.com/ConsenSys/quorum-dev-quickstart/blob/master/files/goquorum/README.md>

The material in these two places don't quite line up - should we combine?
Keeping the docs on the repo helps with the duplication issue that @nmvalera brings up and may be easiest for devs to update.

Reply:
I think the readme in the repo itself can go with just a link to the main docs. Don't know if people use it, but in any case a plain link to the main docs will suffice